shuup/front/templates/shuup/front/macros/navigation.jinja

```diff
@@ -31,7 +32,7 @@
     {% endif %}
 {% endmacro %}
 
-{% macro render_info_dropdown() %}
+{% macro render_info_dropdown(show_quick_login) %}
```

Comment:
make this param optional. we don't know if someone depends on this.

Reply:
fixed